### PR TITLE
Removing links

### PR DESCRIPTION
**Pull request by tushar-hashicorp**
## Changes proposed in this PR:
-
-

## Documentation
- [AWS Standard](<Link the AWS standard which is supported by this policy>)
- [Policy details](<Link the heading to the policy present in the internal CIS policies reference document>)

## Checklist:
- [ ] Tests added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,28 @@
-# Sentinel Policies for the CIS AWS Foundations Benchmark
+# Pre-written Sentinel Policies for AWS CIS Foundations Benchmarking
+
+Pre-written Sentinel policies for AWS CIS Benchmarking provide ready-to-use compliance checks to help ensure your AWS resources meet industry security standards.
```
Comment (absolutelightning) [inline README.md:3]:
```suggestion
Pre-written Sentinel policies are ready to use compliance checks for CIS AWS Foundations Benchmarking v1.2, v1.4 and v3.0 to help enable your AWS resources meet industry security standards.

```

File: README.md [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
+## Feedback
+
+We aim to validate the effectiveness of our policies by collecting diverse user feedback and understanding real-world use cases. This input will help refine our policies and enhance their overall impact. 
+
+1. You can submit your feedback via a [public beta survey](https://docs.google.com/forms/d/e/1FAIpQLScswwLMaVaRuYRGJzDjNiycwM4BUa_gAIsAE_zOPdgyFeLXCA/viewform).
+
+2. If you have any issues or enhancement suggestions to the library, please create [a new GitHub issue](https://github.com/hashicorp/policy-library-cis-aws-cloudtrail-terraform/issues/new).
 
-## Table of Contents
+3. Alternatively, we welcome any contributions that improve the impact of this library! To learn more about contributing and suggesting changes to this library, refer to the [contributing guide](https://github.com/hashicorp/policy-library-cis-aws-cloudtrail-terraform/blob/main/CONTRIBUTING.md).
```
Comment (absolutelightning) [inline README.md:19]: the link will be for this repo right? not for cloudtrail

File: README.md [hunk 3]
```diff
@@ -74,8 +65,8 @@ The documentation for the policy libraries consists of Markdown files within thi
 | Location       | Filename           | Description                            |
 |----------------|--------------------|----------------------------------------|
 | /              | README.md          | Index page for the policy library.     |
-| docs/policies/ | <policy name\>.md | Description of a single policy.        |
-| docs/modules/  | <module name\>.md | Description of a single policy module. |
+| docs/policies/ | <policy name\>.md  | Description of a single policy.        |
```
Comment (absolutelightning) [inline README.md:68]: This is missing - 

```
If you want to change the[ enforcement levels](https://developer.hashicorp.com/sentinel/docs/concepts/enforcement-levels) to either soft-mandatory or hard-mandatory, we recommend updating the contents of the sentinel.hcl file in each library before applying the Terraform configuration.
```

Please check the doc given by product team, after the `Example` section.

**Review by absolutelightning — CHANGES_REQUESTED** [review]

**Review by Nagateja2402 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,17 +1,28 @@
-# Sentinel Policies for the CIS AWS Foundations Benchmark
+# Pre-written Sentinel Policies for AWS CIS Foundations Benchmarking
```
Comment (absolutelightning) [inline README.md:1]:
```suggestion
# Pre-written Sentinel Policies for Center for Internet Security(CIS) AWS Foundations Benchmark
```

**Review by absolutelightning — APPROVED** [review]
One minor suggestion.

**Pull request merged**